apps/src/componentLibrary/tabs/_Tab.tsx

```diff
@@ -51,6 +54,25 @@ const checkTabForErrors = (
   }
 };
 
+const renderButtonContent = (
```

Comment:
Nit: maybe rename as 'renderTabContent' and below 'tabContent' and 'tabElement'?

Reply:
I personally prefer `renderTabButtonContent` but not a strong opinion so good either way.